packages/dx-scheduler-core/src/plugins/drag-drop-provider/helpers.test.ts

```diff
@@ -148,7 +142,167 @@ describe('DragDropProvider', () => {
     });
   });
 
-  describe('#calculateAppointmentTimeBoundaries', () => {
+  describe('#timeBoundariesByResize', () => {
+    it('should not resize if appointment type and cell type are equal', () => {
```

Comment:
cell type -> cell interval?

Reply:
No, I mean cell type. Cell types are `vertical` and `horizontal`

Reply:
then 'are equal' -> 'are different' :)

Reply:
Stop.
Appointment type: 'horizontal' | 'vertical'
Cell type: 'horizontal' | 'vertical'

In this case these types are equal.

Reply:
No, you are right. Different types